src/main/webapp/style.css

```diff
@@ -0,0 +1,17 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap);
+
+:root {
+  --background: #0C0D25;
+  --primary: white;
+  font-size: 16px;
+}
+
+body {
+  background-color: var(--background);
```

Comment:
please use the shorthand property `background` here as well

Reply:
Replaced here b5fc725